test/infamy/netns.py

```diff
@@ -67,11 +67,21 @@ def runsh(self, script, *args, **kwargs):
         return self.run("/bin/sh", text=True, input=script,
                         stdout=subprocess.PIPE, stderr=subprocess.STDOUT, *args, **kwargs)
     
-    def addip(self, addr, subnet=24):
+    def addroute(self, subnet_ip, intermediary, prefix_length=""):
+        if prefix_length:
+            print("Prefix length empty")
+            prefix_length=f"/{prefix_length}"
+        
+        self.runsh(f"""
+            set -ex
+            ip route add {subnet_ip}{prefix_length} via {intermediary}
+            """, check=True)
```

Comment:
- Drop the `print()`, seems to be debug (that's also wrong?)
- Rename `subnet_ip` to just `subnet`
- Rename `intermediary` to `gateway` (or `nexthop`), according to principle of least surprise

Reply:
Thnx :)